examples/customized_welcome_conclusion/custom_conclusion.nsi

```diff
@@ -0,0 +1,69 @@
+Page Custom muiExtraPages_Create
```

Comment:
These example NSIS scripts will be part of the examples and probably consulted by people wanting to implement similar features. It might deserve some documentation on how it works and the overall logic followed. A couple comments at the beginning would be really welcome, at least.